client/app/components/faq/faqdetails.js

```diff
@@ -7,11 +7,11 @@ import {
 import BottomNavigation, { Tab, NavigationComponent } from 'react-native-material-bottom-navigation'
 import Icon from 'react-native-vector-icons/Entypo'
 import { TabNavigator, StackNavigator, HeaderBackButton } from 'react-navigation'
-import { FaqStyles } from '../../styles/faqstyles'
-import { Header } from '../header'
-import { ListSlider } from '../listslider'
+import { FaqStyles } from '../../styles/FaqStyles'
```

Comment:
I got errors here too, looks to be the same problem as the capitalizing of container.js above

Reply:
The files should have been renamed